packages/remix-serve/cli.ts

```diff
@@ -18,7 +18,7 @@ let buildPath = path.resolve(process.cwd(), buildPathArg);
 createApp(buildPath).listen(port, () => {
   let address = Object.values(os.networkInterfaces())
     .flat()
-    .find(ip => ip?.family == "IPv4" && !ip.internal)?.address;
```

Comment:
does the first `ip?.family` check not protect the second?

Reply:
You're right. thanks 🙂